_index.md

```diff
@@ -0,0 +1,104 @@
+# SSS TODO-track-name
+
+This is the public repository for the **TODO-track-name** track of the Security Summer School (SSS).
+It contains information available to participants: session content, slides, activities support files.
+Apart from SSS participants, you can use this content by yourself to learn more about **TODO-track-name**.
+
+## Using the Content
+
+As with all SSS tracks, the **TODO-track-name** track consists of multiple sessions.
+One session typically covers a particular topic of the track.
+Ideally you would go through the sessions orderly, but you can jump ahead to a topic of interest.
+
+Each session has its own folder.
+The `index.md` file from a session folder details the topic of the session and includes practical demos of tools and snippets of code.
+
+After reading the session contents, jump to the hands-on activities.
+A typical activity will challenge you to solve / exploit a given issue and obtain a flag.
+Read the activity description in the session contents and then use the activity support file to get to a solution.
+Each activity will develop your practical skills and help you get a better understanding of the session topics.
+
+Activities come in two forms:
+
+- **tutorials**: that provide a step-by-step detailing of the solution.
+- **challenges**: that only state the goal and provide an initial scaffolding (and maybe some hints), then the rest is up to you.
+  Even so, challenge activity folders store the reference solution for you to take a sneak peak in case you're lost.
+
+## Support
+
+In case you have questions or want to know more about a topic, use the `Discussions` tab or join the [SSS Discord community](https://bit.ly/DiscordSecuritySummerSchool).
+
+## Sessions
+
+The sessions of the SSS **TODO-track-name** track are:
+
+- [Session 01: Name](session-name-1/index.md)
+- TODO
+- TODO
+
+### Session Folder Structure
+
+A session folder consists of:
+
+- `index.md`: session contents, written in [GitHub Markdown](https://guides.github.com/features/mastering-markdown/)
+- `activities/`: each activity has its own folder with support files (including the reference solution)
+- `assets/`: support files for session contents (images, diagrams, code snippets, demos, packet captures, etc.)
+- `slides/`: Markdown file used to generate the session slides.
```

Comment:
This description is inconsistent with those above. And it also doesn't provide a link to the format used (I'm using it's [reveal-md](https://github.com/webpro/reveal-md)).

Reply:
Done.